UnitedIncome/components
# DropDown Portal

### Description
> Please provide a brief description of your changes below.

Creates a portal for the DropDown component, allowing it to appear as a child of an `overflow: hidden` element. This subsequently fixes the animation issue on the itemization screens. 

react-select docs: https://react-select.com/advanced#portaling

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Load up the DropDown on the itemization screen. It will now get portalized and be visible. 

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
